src/app/public/plugin-resources/code-examples/inline-form/custom-buttons/inline-form-demo.component.html

```diff
@@ -30,21 +30,27 @@
 <ng-template
   #inlineFormTemplate
 >
-  <div
-    class="sky-form-group"
+  <form
+    novalidate
+    [formGroup]="myForm"
   >
-    <label
-      class="sky-control-label"
-      [for]="firstName.id"
+    <div
+      class="sky-form-group"
     >
-      First name
-    </label>
-    <input
-      class="sky-form-control"
-      skyId
-      type="text"
-      [(ngModel)]="demoModel.firstName"
-      #firstName="skyId"
-    />
-  </div>
+      <label
+      class="sky-control-label sky-control-label-required"
```

Comment:
Code style, tabs.

Reply:
Wow. Not sure what happened on both of these. Fixed.